svc-bip-api/src/main/java/gov/va/vro/bip/config/RabbitMqConfig.java

```diff
@@ -0,0 +1,43 @@
+package gov.va.vro.bip.config;
+
+import org.springframework.amqp.core.Binding;
+import org.springframework.amqp.core.BindingBuilder;
+import org.springframework.amqp.core.DirectExchange;
+import org.springframework.amqp.core.Queue;
+import org.springframework.amqp.support.converter.Jackson2JsonMessageConverter;
+import org.springframework.amqp.support.converter.MessageConverter;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class RabbitMqConfig {
+  @Value("${abd-data-access.rabbitmq.claim-submit-exchange}")
+  private String claimSubmitExchangeName;
+
+  @Value("${abd-data-access.rabbitmq.claim-submit-queue}")
+  private String claimSubmitQueueName;
+
+  @Value("${abd-data-access.rabbitmq.claim-submit-routing-key}")
+  private String claimSubmitRoutingKey;
```

Comment:
Shouldn't these settings and variable names correspond to the queues mentioned in `BipApiService`?